engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/DeploymentCreateProcessor.java

```diff
@@ -113,8 +111,8 @@ public void processRecord(
       stateWriter.appendFollowUpEvent(key, DeploymentIntent.CREATED, deploymentEvent);
 
       deploymentDistributionBehavior.distributeDeployment(deploymentEvent, key, sideEffects);
-      messageStartEventSubscriptionManager.tryReOpenMessageStartEventSubscription(
-          deploymentEvent, stateWriter);
+      // manage the message, signal, conditional top-level start events subscription
```

Comment:
🤔 This comment is currently not yet correct because we don't yet support signal and conditional events.

🔧 Let's make this future-proof, by changing it to: `manage the top-level start event subscriptions except for timers`